src/style.css

```diff
@@ -0,0 +1,12 @@
+@import url("https://fonts.googleapis.com/css2?family=Bangers&display=swap");
+
+body {
+  font-family: "Bangers", sans-serif;
+}
+
+#content {
```

Comment:
- [ ] Please avoid using ids as selectors in CSS. This is not a good practice to use IDs. Instead, you should use classes.